docs/cloud-native-security/d4c-get-started.asciidoc

```diff
@@ -0,0 +1,59 @@
+[[d4c-get-started]]
+= Get started with CWP
+
+This page describes how to set up Container Workload Protection (CWP) for various use-cases.
+
+[discrete]
+== Initial setup (all use-cases)
+
+First, you'll need to deploy Elastic's Defend for Containers integration to the Kubernetes clusters you wish to monitor.
+
+. Go to *Manage > Container Workload Security > Add Defend for Containers*.
+. Name the integration.
+. Optional — make any desired changes to the integration's policy by adjusting the *Selectors* and *Responses* sections. (For more information, refer to the <<d4c-policy-guide, Defend for Containers policy guide>>). You can also change these later.
+. Under *Where to add this integration*, select an existing or new agent policy.
+. Click *Save & Continue*, then *Add {agent} to your hosts*.
+. On the *Add agent* menu, select the *Kubernetes* tab.
+. Download or copy the manifest.
```

Comment:
We need to add another note to this section due to some late-breaking updates to the elastic YAML

The note should read something like this:

- Edit the `elastic-agent-managed-kubernetes.yml` using your favorite editor
- Remove the comments from the `capabilities` section to allow Defend for Containers the correct permissions

```
            #capabilities:
            #  add:
            #    - BPF # (since Linux 5.8) allows loading of BPF programs, create most map types, load BTF, iterate programs and maps.
            #    - PERFMON # (since Linux 5.8) allows attaching of BPF programs used for performance metrics and observability operations.
            #    - SYS_RESOURCE # Allow use of special resources or raising of resource limits. Used by 'Defend for Containers' to modify 'rlimit_memlock'
```